trade-finance-manager-api/src/graphql/reducers/mappings/gef-facilities/mapGefFacilities.js

```diff
@@ -1,10 +1,13 @@
 const mapGefFacility = require('./mapGefFacility');
 
-const mapGefFacilities = (dealSnapshot, dealTfm) => {
+const mapGefFacilities = async (dealSnapshot, dealTfm) => {
   const { facilities } = dealSnapshot;
 
-  return facilities.map((facility) =>
-    mapGefFacility(facility, dealSnapshot, dealTfm));
+  const mappedFacilities = await Promise.all(facilities.map(async (facility) =>
+    // eslint-disable-next-line no-return-await
```

Comment:
Please remove eslint exception, Lint rules should only be dictated by `eslintrc.config.json`.

Reply:
done